templates/list_pages/node--oe-list-page--oe-w-content-banner.html.twig

```diff
@@ -0,0 +1,12 @@
+{#
+/**
+ * @file
+ * News content banner display.
+ */
+#}
+{{ pattern('content_banner', {
+  title: label,
```

Comment:
since we use the label in the full template perhaps this is where the optional title comes in, do be discussed with business what the optiginal intent for the optional title was